test/studies/shootout/nbody/bradc/nbody-blc.chpl

```diff
@@ -92,21 +91,20 @@ proc initSun() {
 // advance the positions and velocities of all the bodies
 //
 proc advance(dt) {
-  for i in 0..<numBodies {
-    for j in i+1..<numBodies {
-      ref b1 = bodies[i],
-          b2 = bodies[j];
-
-      const dpos = b1.pos - b2.pos,
-            mag = dt / sqrt(sumOfSquares(dpos))**3;
-
-      b1.vel -= dpos * b2.mass * mag;
-      b2.vel += dpos * b1.mass * mag;
+  for param i in 0..<numBodies {
+    for param j in i+1..<numBodies {
+      const dpos = bodies[i].pos - bodies[j].pos,
+            sumOfSq = sumOfSquares(dpos),
+            mag = dt / (sumOfSq*sqrt(sumOfSq));
```

Comment:
Just to make sure I'm understanding this, we've gone from logically:

```chpl
mag = dt / (sqrt(sumOfSq) * sqrt(sumOfSq) * sqrt(sumOfSq))
```

to 

```chpl
mag = dt / (sumOfSq * sqrt(sumOfSq))
```
?

Reply:
Right, though the more I think about this, I feel inclined to back it out.  I think the impact on performance was negligible, if not just noise in testing, and I like that the previous version was more similar to our previous version.  Basically, I think the cleaner code is better than the minor perf boost—if there even was one.  I committed this version because it was literally the version I got my best timings on last night.

Reply:
Just double-checked.  On my Mac, as with my runs last night, I get the best timing with it written in the original commit here:

```
03/24/22        3.151
```

but only by a smidge.  Rewriting the way we do in the currently submitted version, the fastest run I got was:

```
03/24/22        3.178
```

This is close enough to being in the noise compared to the other changes that I think I'll go back to the way we wrote it in the current submission, which seems far more straightforward/less heroic (if it's even having any impact at all).